fcl/caf/cafmaker_defs.fcl

```diff
@@ -193,6 +193,18 @@ caf_ana_sequence: [ mcreco,
     # TODO: rns??
     ]
 
+caf_ana_data_sequence: [
+    # Calorimetry
+    pandoraCaloGausCryoE, pandoraCaloGausCryoW,
```

Comment:
Because calorimetry is not ready, my suggestion would be to exclude the Calorimetry / Low-Energy-Proton-Reco steps from the data chain.

Reply:
I don't necessarily agree we want to turn off the calorimetry outputs, even if the calorimetry constants aren't quite ready. Wouldn't it be better to have this info for testing how things look and just somehow note for the record that these aren't fully finalized? Even if you remove them, calorimetry info will exist for some things, e.g. the showers.